README.md

```diff
@@ -174,20 +174,20 @@ Document | Last Updated | Status |
 
 ## Source code
 
-Compiler and standard library development happens on the `tensorflow` branch of
-the [apple/swift](https://github.com/apple/swift/tree/tensorflow) repository.
+Compiler and standard library development happens on the `main` branch of the
+[apple/swift](https://github.com/apple/swift/tree/main) repository.
 
 Additional code repositories that make up the core of the project include:
 
- - [Swift fork of LLDB](http://github.com/apple/swift-lldb/tree/tensorflow):
-   debugger and REPL support.
  - [Deep learning library](https://github.com/tensorflow/swift-apis): high-level
    API familiar to Keras users.
 
-> Swift for TensorFlow is **not** intended to remain a long-term fork of the official
-> Swift language. Language additions are designed to fit with the direction of
-> Swift and will go through the [Swift
-> Evolution](https://github.com/apple/swift-evolution) process.
+> Swift for TensorFlow is **no longer** a fork of the official Swift language;
+> development was previously done on the `tensorflow` branch of the
+> [apple/swift](https://github.com/apple/swift/tree/tensorflow) repository.
+> Language additions were designed to fit with the direction of Swift and have
+> gone through the [Swift Evolution](https://github.com/apple/swift-evolution)
```

Comment:
Would "are going through the Swift Evolution process" be more accurate, given that Evolution part of the process is ongoing?

Reply:
Yes, that sounds better. Done!